examples/SimpleMidiExample.cpp

```diff
@@ -117,14 +119,16 @@ void midiCallback(double deltaTime, std::vector<unsigned char> *message, void *p
 
         UserData *userData = static_cast<UserData *>(pUserData);
 
-        if (command == 144) {
+        if ((command & 0xf0) == 144) {
```

Comment:
What is the purpose of adding `0xf0` here, and the `0x0f` two lines below? I'm not very familiar with MIDI...

Reply:
Sorry for the late reply! 

The command byte has two functions. The first is the command, the second is the midi channel it is supposed to send the comand to. The '& 0xf0' will remove channel part from the byte and make sure only the command part is checked. I had to add this to be able to support sending commands over multiple midi channels. When you send a midi command over channel 5 for example, the command value will not be 144.